technische-universitat-dresden-historische-musikwissenschaft-note.csl

```diff
@@ -502,7 +504,7 @@
         <text macro="contributors-bib"/>
         <text macro="title-long"/>
         <group>
-          <text macro="container"/>
+          <text macro="container" suffix=", "/>
```

Comment:
Probably better as a delimiter in the group.

Reply:
You are right